autosklearn/pipeline/components/feature_preprocessing/select_rates_classification.py

```diff
@@ -1,29 +1,35 @@
 from ConfigSpace.configuration_space import ConfigurationSpace
 from ConfigSpace.hyperparameters import UniformFloatHyperparameter, \
-    CategoricalHyperparameter, Constant
+    CategoricalHyperparameter
 
 from autosklearn.pipeline.components.base import \
     AutoSklearnPreprocessingAlgorithm
 from autosklearn.pipeline.constants import SIGNED_DATA, UNSIGNED_DATA, SPARSE, DENSE, INPUT
 
 
-class SelectRates(AutoSklearnPreprocessingAlgorithm):
+class SelectClassificationRates(AutoSklearnPreprocessingAlgorithm):
     def __init__(self, alpha, mode='fpr',
                  score_func="chi2", random_state=None):
         import sklearn.feature_selection
 
         self.random_state = random_state  # We don't use this
         self.alpha = alpha
+        self.mode = mode
 
         if score_func == "chi2":
             self.score_func = sklearn.feature_selection.chi2
         elif score_func == "f_classif":
             self.score_func = sklearn.feature_selection.f_classif
+        elif score_func == "mutual_info_classif":
+            self.score_func = sklearn.feature_selection.mutual_info_classif
+            # Work Around as SMAC does not handle Not Equal
```

Comment:
What kind of Not Equal are you referring to? SMAC can have conditions for not being equal: https://automl.github.io/ConfigSpace/master/API-Doc.html#notequalscondition

Reply:
I am referring to:
https://github.com/automl/ConfigSpace/blob/e64aef71a38e0be62fe4fd5b7ea495920fb7f7df/ConfigSpace/read_and_write/pcs.py#L136

Reply:
Oh, that is annoying and we should fix the underlying issue and only output the ConfigSpace as a json. Is SMAC writing the output file or auto-sklearn?

Reply:
We are generating it.

Maybe a WA for the past but we are not using this file.

I am still gonna print it for debugging purposes unless you think otherwise. 

I made a push with this change